package/CHANGELOG

```diff
@@ -55,6 +55,7 @@ Fixes
 Enhancements
  * Introduce parallelization API to `AnalysisBase` and to `analysis.rms.RMSD` class
    (Issue #4158, PR #4304)
+ * explicitly mark `analysis.pca.PCA` as not parallelizable (Issue #4680)
```

Comment:
Newest first

Reply:
I know... but may I respectfully disagree here and suggest that we add anything regarding parallelization *underneath* the parent entry (Issue #4158)? It makes a lot more sense when you read the CHANGELOG.

If it's too confusing to break our convention here, I am happy to switch, of course.

Reply:
No worries